From 4466332db2c82f33e5b0c093c9474195cd876841 Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Thu, 2 Nov 2017 22:26:54 -0400
Subject: [PATCH] One more relaxation for windows

---
 mvpa2/tests/test_hyperalignment.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mvpa2/tests/test_hyperalignment.py b/mvpa2/tests/test_hyperalignment.py
index 053c6d9d..57441ca5 100644
--- a/mvpa2/tests/test_hyperalignment.py
+++ b/mvpa2/tests/test_hyperalignment.py
@@ -253,7 +253,7 @@ class HyperAlignmentTests(unittest.TestCase):
         # not sure yet why on windows only is not precise
         cmp_ = assert_array_equal if (not on_windows) else assert_array_almost_equal
         [cmp_(m.proj, mp.proj) for m, mp in zip(mappers, mappers_nproc)]  # "Mappers differ when using nproc>1."
-        assert_array_equal(ha.ca.residual_errors.samples, ha_proc.ca.residual_errors.samples)
+        cmp_(ha.ca.residual_errors.samples, ha_proc.ca.residual_errors.samples)
         # smoke test
         ha = Hyperalignment(nproc=0)
         mappers = ha(dss_rotated)
-- 
2.15.0

